lib/web_ui/lib/src/engine/text_editing.dart

```diff
@@ -330,6 +330,11 @@ class TextEditingElement {
     switch (inputConfig.inputType) {
       case InputType.text:
         domElement = owner.createInputElement();
+        // Setting the type to password seems to be only option to prevent the obscured text
```

Comment:
Can you add a unit test for testing this configuration?

In test_editing_test.dart (line 20-35) there are 2 input configurations now. You can also add a third one with type text and obscuretext=true. Later use it in a test case.